cdhweb/people/migrations/0011_populate_nonproxy_person.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 2.2.17 on 2020-12-22 16:54
+
+from django.conf import settings
+from django.db import migrations
+
+
+def create_nonproxy_persons(apps, schema_editor):
+    User = apps.get_model('auth', 'User')
+    Person = apps.get_model('people', 'Person')
+    # Profile = apps.get_model('people', 'Profile')
+
+    # iterate over all users (except script user) and create person records
+    # populate new person from user and profile
+    # set user - person 1:1 and save
+    for user in User.objects.exclude(username=settings.SCRIPT_USERNAME):
+        person_info = {
+            'user': user,
+            'first_name': user.first_name,
+            'last_name': user.last_name
+        }
+        # get profile if there is one (not all users have them)
+        profile = getattr(user, 'profile', None)
+        if profile:
+            person_info.update({
+                'cdh_staff': profile.is_staff,
+                'job_title': profile.job_title,
+                'department': profile.department,
+                'institution': profile.institution,
+                'pu_status': user.profile.pu_status
+            })
+        Person.objects.create(**person_info)
+
+    # NOTE: prune users who don't have login access? later?
+
+    # TODO: do we care about log entries for person creation?
+    # get script user and log entry model to document changes
+    # script_user = User.objects.get(username=settings.SCRIPT_USERNAME)
+    # LogEntry = apps.get_model('admin', 'LogEntry')
+    # ContentType = apps.get_model('contenttypes', 'ContentType')
```

Comment:
I think we sectioned off all the log entry creation stuff into #247, although looking at it, that issue only mentions Pages. maybe we should expand it?

Reply:
IIRC #247 is about transferring existing log entries when we exodize records; this todo is about (potentially) creating log entries to document that these person entries were created via script, which I think needs to be done here (if we do it)

Reply:
gotcha. that seems simple enough, I'll give it a shot - agree we should do it

Reply:
Great. I think I have examples elsewhere if it would be helpful

Reply:
Added log entry creation, but I noticed that there's no way to view them in the wagtail admin (that I can find). Not sure how important this is though since we can do it in console. I modified `admin.py` so it would work enough to go in and verify that they are indeed there.